src/erasure.rs

```diff
@@ -536,7 +538,10 @@ pub fn recover(
         );
     }
 
-    assert!(!corrupt);
+    if corrupt {
```

Comment:
cool, but we want to remove or mark as bogus the coding blobs from the ledger?  maybe we need a way to say "this is a bogus blob" in the ledger/window...

Reply:
yeap.